in-depth/submission_guidelines.md

```diff
@@ -86,6 +86,10 @@ Heading
 
 : The name of your theme or plugin.
 
+View
+
+: Choose between the image oriented `jumbovision` layout, or the text-oriented `card` layout.  The default for Themes is `jumbovision`, while plugins use `card`.  This option only needs to be specified if you'd like to use the opposite layout.
```

Comment:
However, mentioning how to change the button's text definitely makes sense :smiley: